api/orchestrator/orchestrator.go

```diff
@@ -62,6 +63,12 @@ type CloudServiceRequest interface {
 	proto.Message
 }
 
+type LogRequest interface {
+	GetId() string
```

Comment:
Maybe "target ID" / payload ID oder something similar? Its not really the ID of the request but rather of its payload/the target.